src/GraphNodeManagerViewExtension/ViewModels/GridNodeViewModel.cs

```diff
@@ -502,6 +527,9 @@ private void EvaluateNode(NodeModel nodeModel, string propertyName = "Modified")
                 case "State":
                     RaisePropertyChanged(nameof(State));
                     break;
+                case "DismissedAlertsCount":
```

Comment:
can't you also use nameof for the cases here?

Reply:
Yup, done!